modules/admin-server/src/main/resources/db/migration/V0.1.0__Account.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TYPE "public"."KMS_TYPE" AS ENUM ('LOCAL');
+
+CREATE TABLE "public"."account" (
```

Comment:
we should not hardcode schema "public". It's up to the config file to set the schema name

Reply:
Removed public from the table name